sentry-core/src/main/java/io/sentry/core/protocol/SentryStackFrame.java

```diff
@@ -18,8 +19,13 @@
   private String absPath;
   private String contextLine;
   private Boolean inApp;
-  private String _package; // TODO: _package as its a reserverd word
-  private Boolean _native; // TODO: _native as its a reserverd word
+
+  @SerializedName(value = "package")
```

Comment:
@bruno-garcia the solution here is not ideal, an adapter would be better, but I'd like to fix all the open bugs asap.
later could be replaced with an adapter without a breaking change.

Reply:
Wfm